File: pset2_Arrays/scores.c
#include <stdio.h>
#include <cs50.h>

// 數組

int main(void)
{
    const int N = 3; //設定常數固定為3
    int scores[N]; //定義數組
    for(int i = 0; i < N; i++)
    {
        scores[i] = get_int("Scores: "); //詢問常數次數 分數
    }

    // print average
    printf("Average: %f\n",(scores[0] + scores[1] + scores[2]) / (float) N); //數組相加後 / (浮點數) 常數 3

}

File: pset6_python/scores.py
from cs50 import get_int

scores = []

for i in range(3):
    score = get_int("Score: ")
    # scores += [score]
    scores.append(score) # .append 增加到串列

average = sum(scores) / len(scores)

print(f"Average: {average}")
